src/components/Calculator.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
- [ ] I think it could be a good idea to add some negative tests, this will help us be sure that in any kind of scenario our application will behave correctly, so kindly let's try to simulate negative scenarios as well in our tests 😉